content/js1/blocks/assertions/index.md

```diff
@@ -65,25 +65,21 @@ Predict and explain if the assertion will succeed or fail. Pay particular attent
 
 ### Clarity with arguments
 
-It would be useful to have more information as to why this assertion failed. We can pass additional arguments to `console.assert`:
+It would be useful to have more information as to why this assertion failed. We can pass an additional argument to `console.assert`:
 
 ```js
 function formatAs12HourClock() {}
 
 console.assert(
   formatAs12HourClock("08:00") === "08:00 am",
-  "current function output: %s, target output: %s",
-  formatAs12HourClock("08:00"),
-  "08:00 am"
+  `current output: ${currentOutput}, target output: ${targetOutput}`
```

Comment:
This one seems broken? It dropped the arguments, and references variables that don't exist?

We either need to extract a variable, or undo this change?

Reply:
I just updated this assertion so there's no reference to any variables prior to the refactor section. Should all be in order now.